### ISSUE TITLE
Change payload from string to json object in Mongo

### ISSUE THREAD

**Issue by kx499** [threatstream/mhn#174]
I was playing this over the weekend. Is there any appetite for this? I feel like it would make queries and searches on that data more efficient. 

I made the changes in mnemosyne so it logs to hpfeed collection with the json payload, then updated the normalizer modules (all on my dev version). I was kind of bored.

**Comment by aabed** [comment]
I don't see a pull request , Also I have opened your github and found no changes over there

**Comment by jt6211** [comment]
yes, i think we would be open to this.  I wasn't initially, but I've heard of a couple people make this change and it hasn't caused them any trouble

**Comment by kx499** [comment]
I've only made the changes to Mnemosyne install I have locally. I'll fork it and push the changes tonight. I have still yet to make the changes to mhn, will do so tonight as well.

**Comment by kx499** [comment]
Pushed the changes to my fork. I'm still playing with some things, but feel free to check it out. 

Looking into some performance issues with the payloads page before i do a PR

**Comment by cloughrm** [comment]
@kx499 any update on this issue?

**Comment by d1str0** [comment]
@kx499 still interested in contributing for this issue? Any updates?
